src/api/todos.ts

```diff
@@ -0,0 +1,28 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = (data: Omit<Todo, 'id'>) => {
+  return client.post('/todos', data);
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const updateTodoComplited = (
```

Comment:
Fix the typo in the function name `updateTodoComplited` to `updateTodoCompleted`.